backend/app/testbot/kb.py

```
@@ -104,34 +104,40 @@ def load_file_index(chatbot_id: str, file_path: str) -> Tuple[Optional[Any], Opt
     """
-    Discover all supported files in the data directory and build indexes for each
-    Returns a dictionary of {file_path: (index, documents)}
+    Discover all supported files in the data directory and build indexes for each.
+    Returns a dictionary of {file_path: (index, documents)}.
     """
     indexes = {}
-    
-    # Make sure the directory exists
+
     if not os.path.exists(data_dir):
         logger.warning(f"Data directory {data_dir} does not exist")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)